testsuite/tests/typing-misc/external_arity.ml

```diff
@@ -0,0 +1,33 @@
+(* TEST
+   * expect
+*)
+
+let f a b c = a + b + c
+
+let _ : int = Obj.magic f None None None
```

Comment:
Out of curiosity, what is this part of the test for?

Reply:
See https://github.com/ocaml/ocaml/pull/2015#issuecomment-418109349.